src/components/Task/Modal/Form/Inputs/ActivityTypeAutocomplete/ActivityTypeAutocomplete.test.tsx

```diff
@@ -18,13 +16,15 @@ describe('ActivityTypeAutocomplete', () => {
 
   loadConstantsMockData?.constant.phases?.forEach((phase) => {
     phase?.tasks?.forEach((task) => {
+      const activity = loadConstantsMockData?.constant?.activities?.find(
+        (activity) => activity.id === task,
+      );
       activitiesMap.set(task, {
-        name: getLocalizedTaskType(i18n.t, task),
+        translatedFullName: activity?.value,
+        translatedShortName: activity?.action,
```

Comment:
Everywhere we use either of these fields we have to do `|| ''`. What do you think about making the type `string` and setting the default value here?

```suggestion
        translatedFullName: activity?.value || '',
        translatedShortName: activity?.action | '',
```